/**:
  traversability_node:
    ros__parameters:
      additions_topic_name: "traversability_keyframe_additions"
      updates_topic_name: "traversability_keyframe_updates"
      pointcloud_topic_name: "lidar/points"
      use_lidar_pointcloud: false
      publish_traversability_grid: true

  local_traversability_node:
    ros__parameters:
      pointcloud_topic_name: "/fused_pointcloud"
      expected_frequency: 4.0 # Hz.

  traversability_threshold_node:
    ros__parameters:
      lethal_obstacle_threshold: 70 # between 0 to 100